client/verta/verta/credentials.py

```diff
@@ -113,16 +122,19 @@ def headers(self):
             # without underscore, for NGINX support
             # https://www.nginx.com/resources/wiki/start/topics/tutorials/config_pitfalls#missing-disappearing-http-headers
             "bearer-access-token": self.jwt_token,
+            "organization-id": self.organization_id or "",
```

Comment:
Since we are checking for and adding the org id below, we could maybe just drop this line.